src/templates/readme.ejs

```diff
@@ -1,47 +1,55 @@
-# What's Herbs?
+# <%- props.name %>
+<%- props.description %>
 
-Herbs is an open source library for backend applications. It is focused on achieving faster deliveries and with happier developers, without neglecting the long-term need to constantly evolve your application as requirements change.
+## Template
+This is a simple readme template generated by herbs cli to guide you through a great and domain driven documentation for your project.
+✅ Domain driven
+❌ Technical details
 
-See more on the website: [https://herbsjs.org/](https://herbsjs.org/)
+## What it does
+It should briefly capture the main reason why this system exists.
+✅ This microservice manages all the recurrent and usage based billing for the company.
+❌ This microservice uses Postgres 14.23 and some message queue to handle billing.
 
-# Getting Started with CLI on Herbs
+## What it doesn't do (optional)
+It is important to explain here if it is common for people to misunderstand the system purpose and its features 
+✅ This system doesn't process payments, it only generate bills. Check the Payment System for that.
 
-Within this repository, you will find a front-end application, made in react consuming a back-end application, using [Herbs](https://github.com/herbsjs), the API is made in graphQL or rest (this will depend on what you chose during project creation)
+## Main features
+It should be a list of top 3 or 5 features or use cases.
+✅ - Products list (CRUD)
+✅ - Product usage billing management
+✅ - Product recurring billing management
+✅ - Product billing closing
+✅ For more features: [link to project's Herbs Shelf]
 
-# Herbs CLI
+❌ - REST and GraphQL APIs
+❌ - Serverless
+❌ - Handles 1000 transactions/second
 
-This is a example on how to build a backend application using [Herbs](https://github.com/herbsjs).
+## Using
+To start the project for the first time:
 
-### Using
+```
+$ npm install
+$ npm run knex:migrate
+$ npm start
+```
 
-    $ npm install
-    $ npm run knex:migrate (in case of using relational database with herbs2knex, not necessary for MongoDB)
-    $ npm start
+You should receive this message => 🚀 Server UP and Running in port: 3000 (this will depend on what you chose during project creation).
```

Comment:
The message from `npm start` should be self explanatory so no extra info is necessary. This text can be removed